packages/editor-ui/src/components/MainHeader/WorkflowDetails.vue

```diff
@@ -497,12 +501,17 @@ export default defineComponent({
 				cb(true);
 				return;
 			}
-
-			const saved = await this.workflowHelpers.saveCurrentWorkflow({ name });
-			if (saved) {
-				this.isNameEditEnabled = false;
+			try {
+				this.uiStore.addActiveAction('workflowSaving');
+				const saved = await this.workflowHelpers.saveCurrentWorkflow({ name });
+				if (saved) {
+					this.isNameEditEnabled = false;
+				}
+				// Exception is handled in the helper function
```

Comment:
Ah I didn't realize that `saveCurrentWorkflow` already handles exceptions 🙈 In that case the `try..finally` is probably not needed. Feel free to remove it

Reply:
Yeah, I was a bit surprised by `try` request but thought you saw something I didn't :D Let's remove it since it's cleaner without it